.npmignore

```diff
@@ -1,5 +0,0 @@
-# Remove ES6 source
-src/
```

Comment:
@esakal are you sure publish will work probably and not include the raw, non-transpiled js files?

Reply:
yes, I took a more strict approach by mentioning what *should* be included instead of what *shouldn't* using [package.json > files options](https://docs.npmjs.com/files/package.json#files)

https://github.com/eshaham/israeli-bank-scrapers/blob/50873491dcdf07142b3fd923cedfa176db5166e3/package.json#L58-L60

Yet I encourage you to pack everything `npm pack` and verify that your local test configuration and  others like `src` are not included as expected

Reply:
@esakal 👍